lib/fork.js

```diff
@@ -3,8 +3,27 @@
 const childProcess = require('child_process')
     , childModule  = require.resolve('./child/index')
 
+let checked, available
+function fork (forkModule, workerOptions, worker_threads) {
+
+  // Check whether we need to run using worker_threads.
+  if (worker_threads) {
```

Comment:
I get why you've gone with this name but I think I'd prefer it be `workerThreads`